static/js/cohort_filelist.js

```diff
@@ -35,7 +36,8 @@ require.config({
         'jqueryui': ['jquery'],
         'session_security': ['jquery'],
         'assetscore': ['jquery', 'bootstrap', 'jqueryui'],
-        'assetsresponsive': ['jquery', 'bootstrap', 'jqueryui']
+        'assetsresponsive': ['jquery', 'bootstrap', 'jqueryui'],
+        'tokenfield': ['jquery', 'jqueryui']
```

Comment:
Is this using the tokenfield like in the gene selection page? I was thinking it should be more like the selected filters on the cohort creation page.